services/galley/test/integration/TestSetup.hs

```diff
@@ -1,61 +1,26 @@
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# OPTIONS_GHC -fprint-potential-instances #-}
 module TestSetup
-    ( test
-    , tsManager
-    , tsGalley
-    , tsBrig
-    , tsCannon
-    , tsAwsEnv
+    ( tsAwsEnv
     , tsMaxConvSize
     , TestM(..)
     , TestSetup(..)
     ) where
 
 import Imports
-import Test.Tasty          (TestName, TestTree)
-import Test.Tasty.HUnit    (Assertion, testCase)
-import Control.Lens        (makeLenses, view)
-import Control.Monad.Catch (MonadCatch, MonadMask, MonadThrow)
-import Bilge (Manager, MonadHttp(..), Request)
+import Bilge (Manager)
+import Forecastle
+import Control.Lens (makeLenses)
 
 import qualified Galley.Aws          as Aws
 
-type GalleyR      = Request -> Request
-type BrigR        = Request -> Request
-type CannonR      = Request -> Request
-
 data TestSetup = TestSetup
-    { _tsManager     :: Manager
-    , _tsGalley      :: GalleyR
-    , _tsBrig        :: BrigR
-    , _tsCannon      :: CannonR
+    { tsManager     :: Manager
```

Comment:
I'm no longer generating lenses for these since it's probably better to use the versions from Forecastle; but it's kinda weird to only generate lenses for some fields and not others haha.